modules/collect/src/main/java/com/opengamma/strata/collect/array/DoubleArray.java

```diff
@@ -263,19 +264,19 @@ public static DoubleArray ofUnsafe(double[] array) {
 
   //-----------------------------------------------------------------------
   /**
-   * Obtains an instance from a list of {@code Double}.
+   * Obtains an instance from a collection of {@code Double}.
```

Comment:
Because order matters in an array, but not in a collection, we should document the order assunption